### ISSUE TITLE
Index files with unsupported file endings cause a KeyError

### ISSUE THREAD

**Issue by Anteru** [Anteru/liara#2]
Steps to reproduce:

```
$ liara quickstart
$ liara mv content/_index.md content/_index.md~
```

**Comment by Anteru** [comment]
The underlying issue here is that `_index` nodes don't go through the document factory check if the factory even supports the file type.

**Comment by Anteru** [comment]
Fixed by ecafff1d0055e332959b6af503be9dfa85af9e73
